src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -57,18 +58,21 @@ static ucs_status_t uct_cuda_copy_iface_query(uct_iface_h iface,
     iface_attr->device_addr_len         = 0;
     iface_attr->ep_addr_len             = 0;
     iface_attr->cap.flags               = UCT_IFACE_FLAG_CONNECT_TO_IFACE |
+                                          UCT_IFACE_FLAG_GET_SHORT |
+                                          UCT_IFACE_FLAG_PUT_SHORT |
                                           UCT_IFACE_FLAG_GET_ZCOPY |
                                           UCT_IFACE_FLAG_PUT_ZCOPY |
                                           UCT_IFACE_FLAG_PENDING;
 
-    iface_attr->cap.put.max_short       = 0;
+    iface_attr->cap.put.max_short       = INT_MAX;
```

Comment:
why not UINT_MAX?

Reply:
it can.  choose this because this will be used in  small message protocols and also wanted to restrict cuda allocation size in gtest.

Reply:
then need to add length check in PUT_SHORT..

Reply:
you mean in function uct_cuda_copy_ep_put_short() ?

Reply:
maybe i should change it to UINT_MAX instead of additional check in the func